afuc/afuc.h

```diff
@@ -178,4 +178,149 @@ afuc_set_opc(afuc_instr *ai, afuc_opc opc, bool rep)
 	}
 }
 
+typedef struct {
+	unsigned num;
+	const char *name;
+} afuc_config_reg;
+
```

Comment:
creating a new xml file might be overkill..  but just adding an enum (maybe keyed of gpu generation, like `a6xx_config_regs`) into the existing xml shouldn't be too bad.  And might be a bit faster to iterate as new config regs are named.. and we already use this for CP packet names?  I suppose the body of `getpm4()` could be extracted out into a helper, and `find_enum_val()` re-used..

Reply:
I did basically that... `getpm4()` had some extra handling to deal with with variants re-using values which isn't useful here (pretty much everything gets changed around AFAICT, so we use separate enums) so I didn't reuse it though.